requirements.txt

```diff
@@ -1 +1,2 @@
--e .
+setuptools==48.0.0
```

Comment:
Pinning to older version because new version does not support `.` in requirements.txt

Reply:
Can you update https://github.com/instacart/lore/blob/master/lore/dependencies.py if this is a real dependency?